File: solutions/uri/1005/1005.clj
(ns main)

(defn main []
  (let [a (Double/parseDouble (read-line))
        b (Double/parseDouble (read-line))]
    (format "%.5f" (/ (+ (* a 3.5) (* b 7.5)) 11.0))))

(println "MEDIA =" (main))

File: solutions/uri/1005/1005.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);

    val a: String = sc.next();
    val b: String = sc.next();

    println("MEDIA = %.5f".format((a.toDouble() * 3.5 + b.toDouble() * 7.5) / 11.0));
}
